public/assets/link.svg

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I'd recommend running `[svgo](https://github.com/svg/svgo) --disable=removeViewBox public/assets/link.svg` on these files to remove unnecessary cruft.

Reply:
that is being done automatically by ember-svg-jar :)